README.md

```diff
@@ -1,20 +1,28 @@
-## Write the Docs website
+# ✍️ Write the Docs website
 
-This is the code that powers [www.writethedocs.org](http://www.writethedocs.org). It contains information about the Write the Docs group, as well as information about writing documentation.
+🔵 This is the code that powers [www.writethedocs.org](http://www.writethedocs.org). It contains information about the Write the Docs group, as well as information about writing documentation.
 
-To contribute to the Write the Docs website, it's helpful to familiarize yourself with the [Sphinx site generator](https://www.sphinx-doc.org/), as well as [reStructuredText markup syntax](https://www.sphinx-doc.org/en/stable/rest.html).
+🔵 To contribute to the Write the Docs website, it's helpful to familiarize yourself with the [Sphinx site generator](https://www.sphinx-doc.org/), as well as [reStructuredText markup syntax](https://www.sphinx-doc.org/en/stable/rest.html).
 
-### Code Architecture
+## ⬅ Prerequisites for generating the docs locally
+
+1. Install `python 3.8.x` using your package manager, if not installed already. You'll probably need `root` privileges to do this.
```

Comment:
It's not exclusively 3.8 though - I think we support 3.7 and up?

Reply:
But it keeps the instructions more straightforward. If this is the only issue, let's get this merged?

Reply:
I'm gonna merge this :-D